source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -61,38 +66,38 @@
 }
 
 
-def geojson_argument(value):
-    decoded = json.loads(value)
-    if not decoded:
-        raise ValueError('invalid shape')
+class geojson_argument(object):
+    def __call__(self, value):
+        decoded = json.loads(value)
+        if not decoded:
+            raise ValueError('invalid shape')
+
+        return parser_args_type.geojson_argument(decoded)
+
+    def description(self):
+        return ParameterDescription(type=str)  # TODO a better description of the geojson
```

Comment:
hum I don't think we can do this, I don't see how the SDK will be able to handle this parameter (but maybe we don't need it and we can do a simple thing as giving a link to the documentation as you said)